src/contour/shell-integration/shell-integration.fish

```diff
@@ -0,0 +1,46 @@
+# vim:et:ts=4:sw=4
+#
+#/**
+# * This file is part of the "contour" project
+# *   Copyright (c) 2019-2021 Christian Parpart <christian@parpart.family>
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# */
+
+# Example hook to change profile based on directory.
+#function update_profile()
+#    switch "$PWD"
+#        case "$HOME"/work* 
+#            contour set profile to work
+#        case "$HOME"/projects* 
+#            contour set profile to main
+#        case '*' 
+#            contour set profile to mobile
+#    end
+#end
```

Comment:
I think this is POSIX syntax here, right? then we should probably get rid of it or update to be fish syntax.